runtime/src/main/cpp/Memory.cpp

```diff
@@ -2254,6 +2313,18 @@ KInt getGCThreshold() {
   return memoryState->gcThreshold;
 }
 
+void setGCCollectCyclesThreshold(KInt value) {
+  GC_LOG("setGCCollectCyclesThreshold %d\n", value)
+  if (value > 0) {
```

Comment:
I think it's better to throw `IllegalArgumentException` here (when it's called from kotlin user code; when it gets called from runtime on it's own behalf, it should `RuntimeAssert`)